admin_manual/appliance/certificates.rts

```diff
@@ -0,0 +1,24 @@
+=======================
+How to add certificates
+=======================
+
+If you want to add your own certificates for the appliance,
+
+you will have to follow this steps.
+
+1. Create the certificates and deposit them on your appliance.
+2. Connect to your appliance
+  - directly on the command line of your virtual machine
+  - via ssh connection to your appliance
+3. execute this 2 commands
+
+ucr set apache2/ssl/certificate="/etc/myssl/cert.pem"
+ucr set apache2/ssl/key="/etc/myssl/private.key"
+
+Adjust the path and filename to match your certificate.
+
+then restart the apache2 service
```

Comment:
s/then restart the apache2 service/Then restart the apache2 service, using the following command:/g